pipeline/ingest/histology.py

```diff
@@ -154,10 +153,16 @@ def _load_histology_ccf(self):
 
             for probepath, shank_no in zip(probefiles, shanks):
                 hist = scio.loadmat(probepath, struct_as_record=False, squeeze_me=True)['site']
+                px_res = int(hist.pos.mmPerPixel * 1000)
```

Comment:
nice catch - was previously assuming input at a fixed resolution (also why I deliberately didn't round since the code assumed an integer multiple of the input data)